var/spack/repos/builtin/packages/superlu/package.py

```diff
@@ -34,21 +35,86 @@ class Superlu(Package):
     url      = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_5.2.1.tar.gz"
 
     version('5.2.1', '3a1a9bff20cb06b7d97c46d337504447')
+    version('4.3', 'b72c6309f25e9660133007b82621ba7c')
 
-    depends_on('blas')
+    variant('blas', default=True, description='Build with external BLAS library')
```

Comment:
I share @davydden opinion on having packages sneaking dependencies in, in particular for libraries. If we start to do this, there's no way spack can ensure the correctness of a build.